runtime/src/chpl-qsbr.c

```diff
@@ -87,42 +87,52 @@ struct tls_node {
 static struct tls_node *chpl_qsbr_tls_list;
 static CHPL_TLS_DECL(struct tls_node *, chpl_qsbr_tls);
 
+static inline struct tls_node *get_tls_list();
```

Comment:
I wouldn't expect you'd need a separate prototype for a 'static inline' function. What compiler was giving you an error?

Reply:
It was Travis, which has `-Werror=strict-prototypes` set

Reply:
Oh wait, I forgot that it has to have void for parameter list!